src/bot.ts

```diff
@@ -99,7 +99,13 @@ export class DiscordBot {
         return this.channelSync;
     }
 
-    public GetIntentFromDiscordMember(member: Discord.GuildMember | Discord.User): Intent {
+    public GetIntentFromDiscordMember(member: Discord.GuildMember | Discord.User, webhookID?: string): Intent {
+        if (webhookID) {
+            // webhookID and user IDs are the same, they are unique, so no need to prefix _webhook_
+            const name = member instanceof Discord.User ? member.username : member.user.username;
+            // no need to escape the mxid as getIntentFromLocalpart does that alreaddy down the road
+            return this.bridge.getIntentFromLocalpart(`_discord_${webhookID}_${name}`);
```

Comment:
Oh eer, you need to do ```new Matrix(`_discord_${webhookID}_${name}`).userId```

Reply:
*MatrixUser even

Reply:
It does that under the hood, soru checked the library extra

Reply:
Okay, if you are sure :)

Reply:
actually wait